pkg/network/tracer/connection/fentry/tracer.go

```
@@ -51,6 +53,24 @@ func LoadTracer(config *config.Config, m *manager.Manager, mgrOpts manager.Optio
 		telemetryMapKeys := errtelemetry.BuildTelemetryKeys(m)
 		o.ConstantEditors = append(o.ConstantEditors, telemetryMapKeys...)
 
+		file, err := os.Stat("/proc/self/ns/pid")
+
+		if err != nil {
+			return nil
```

I would return an error here.